### PR TITLE
Add recipe for the R package secuTrialR

### PR DESCRIPTION
**Pull request by PatrickRWright**
One question: Is there some kind of citation for conda-forge that I can put in a publication?

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by PatrickRWright** [comment]
I am no expert so I prefer asking: Is what I did in the `test_alternative` commit ok? If it is then it looks like its all working as suggested.

I was wondering, however, if I should rather use the following types of recipes as templates:
https://github.com/conda-forge/r-seqinr-feedstock/tree/master/recipe
https://github.com/conda-forge/r-transport-feedstock/tree/master/recipe

Some feedback would be appreciated. Thanks.

**Comment by xhochy** [comment]
Best is to generate R recipes using https://github.com/bgruening/conda_r_skeleton_helper

**Comment by PatrickRWright** [comment]
Thanks @xhochy for the reply. I saw that helper, but from the documentation my impression was that it's just for packages that are on CRAN.

**Comment by xhochy** [comment]
Oh, yes! Then this recipe looks good!

File: recipes/r-secutrialr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+about:
+  home: https://swissclinicaltrialorganisation.github.io/secuTrialR/
+  dev_url: "{{ github }}"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
+  summary: Handling of Data from the Clinical Data Management System secuTrial 
+
+extra:
+  recipe-maintainers:
```
Comment (xhochy) [inline recipes/r-secutrialr/meta.yaml:75]: Would be good to have `conda-forge/r` as a maintainer, too. I'll add them once the feedstock is generated.

Comment (PatrickRWright) [reply recipes/r-secutrialr/meta.yaml:75]: Thank you, its appreciated.

**Comment by PatrickRWright** [comment]
One more question @xhochy : Is there some kind of publication I can reference with regard to conda-forge?

**Comment by xhochy** [comment]
Not that I'm aware of but feel free to open an issue at https://github.com/conda-forge/conda-forge.github.io asking for this. There is the most fitting place to ask this.

**Pull request merged**